packages/web/src/scss/theme/_form-fields.scss

```diff
@@ -49,10 +51,17 @@ $box-field-variants: (
         input-border-color: $box-field-input-border-color-warning,
         message-color: $box-field-message-color-warning,
     ),
-    error: (
-        input-border-color: $box-field-input-border-color-error,
-        message-color: $message-color-error,
+    danger: (
+        input-border-color: $box-field-input-border-color-danger,
+        message-color: $message-color-danger,
     ),
+
+    // @deprecated Will be removed in next major version.
+    error:
```

Comment:
can we format this the same way as in danger or is this a prettier thing?

Reply:
probably prettier thing